content/cloud/linking-applications-to-resources-using-labels.md

```diff
@@ -0,0 +1,152 @@
+- Create a new database and link the app to it
+
+In this example below, we are going to **link** the app to an existing NoOps SQL Database instance using the label. To do this, we choose the first option (that `spin deploy` gives us). We make our selection by using the up/down arrow keys and then `return` to continue:
+
+```bash
+spin deploy
+Uploading todo-app version 0.1.0-r234fe5a4 to Fermyon Cloud...
+Deploying...
+App "todo-app" accesses a database labeled "data"
+Would you like to link an existing database or create a new database?:
+> Use an existing database and link app to it
+  Create a new database and link the app to it
+```
+
+Since we chose to link to an existing database, `spin deploy` asks us which database we would like to link this app's "data" label to:
+
+```bash
+Which database would you like to link to todo-app using the label "data":
+  finance
+> projects
+```
+
+> Note: When Fermyon Cloud creates a new databases, it gives it an arbitrary name such as `inspiring-otter`. You can rename databases to give them more memorable names. In the example above, our databases have been renamed for `finance` and `projects`.
+
+From this point, we see that our app is deploying and once finished we receive our deployed application's URL endpoint:
+
+```bash
+Waiting for application to become ready...... ready
+Available Routes:
+  todo-app: https://todo-app-ylnmlqmr.fermyon.app (wildcard)
+```
+
+Let's kick things up a notch and deploy a second Spin application (`todo-app-v2`) that will also use the same Cloud database:
+
+```toml
+// --snip--
+name = "todo-app-v2"
+[component]
+id = "todo"
+sqlite_databases = ["default"]
+// --snip--
+```
+
+When deploying the second Spin application (`todo-app-v2`) we are again asked if we would like to use an existing database and link our application (in this case `todo-app-v2`) to it:
+
+```bash
+spin deploy       
+Uploading todo-app-v2 version 0.1.0-r6e577b18 to Fermyon Cloud...
+Deploying...
+App "todo-app-v2" accesses a database labeled "default"
+Would you like to link an existing database or create a new database?:
+> Use an existing database and link app to it
+  Create a new database and link the app to it
+```
+
+We again use the up/down arrow keys to make our selection and hit `return` to continue:
+
+```bash
+Which database would you like to link to todo-app-v2 using the label "default":
+> inspiring-otter
+```
+
+In the example here we have chosen to link `todo-app-v2` to our pre-existing `inspiring-otter` Cloud database using via the "default" label, that we added to our `todo-app-v2` application manifest (`spin.toml`) file.
+
+Now we have two separate Spin applications (`todo-app` and `todo-app-v2`) using their own labels to link to the same database. 
+
+Links and labels are application scoped. Whenever a specific Spin application (say `todo-app`) needs to access the database resource (`inspiring-otter`), it will do so via the label "data".
```

Comment:
```suggestion
Links and labels are application scoped. Whenever a specific Spin application (say `todo-app`) needs to access the database resource (`projects`), it will do so via the label "data".
```